drivers/wifi/nrf700x/src/wifi_mgmt_scan.c

```diff
@@ -118,6 +118,12 @@ int nrf_wifi_disp_scan_zep(const struct device *dev, struct wifi_scan_params *pa
 		goto out;
 	}
 
+#if defined CONFIG_WIFI_NRF700X_SKIP_LOCAL_MAC
```

Comment:
`scan_info->scan_params.skip_local_mac = IS_ENABLED(CONFIG_WIFI_NRF700X_SKIP_LOCAL_MAC);`

And you can drop the macros too.

Reply:
Done